src/upload/catalog/controller/extension/payment/libwebpay_rest/HealthCheck.php

```diff
@@ -24,97 +21,71 @@ public function __construct($config) {
         $this->apiKey = $config['API_KEY'];
         $this->ecommerce = $config['ECOMMERCE'];
         // extensiones necesarias
-        $this->extensions = array(
-            'openssl',
-            'SimpleXML',
-            'soap',
-            'dom'
-        );
+        $this->extensions  = ['openssl', 'SimpleXML', 'soap', 'dom'];
     }
 
     // valida version de php
-    private function getValidatephp(){
-        if (version_compare(phpversion(), '8.0', '<=') and version_compare(phpversion(), '7.0', '>=')) {
-            $this->versioninfo = array(
-                'status' => 'OK',
-                'version' => phpversion()
-            );
-        } else {
-            $this->versioninfo = array(
-                'status' => 'Error!: Version no soportada',
-                'version' => phpversion()
-            );
-        }
-        return $this->versioninfo;
+    private function getValidatephp(): array{
+        $phpVersion = phpversion();
+        $status = (version_compare($phpVersion, '8.0', '<=') && version_compare($phpVersion, '7.0', '>=')) ? 'OK' : 'Error!: Version no soportada';
```

Comment:
The language of these messages should be standardized, either all in English or Spanish.